packages/shared/styles/components/atoms/SfInput.scss

```diff
@@ -42,6 +42,15 @@
       var(--font-family--secondary)
     );
   }
+  &__placeholder-shown{
+    --icon-color: var(--c-primary);
+    --input-password-icon-background: var(--c-primary);
+    --input-label-top: 0;
+    --input-label-color: var(--c-primary);
+    --input-label-font-size: var(--font-xs);
```

Comment:
should be ```var(--font-size--xs)``` :)

Reply:
But it should be of same size as it is when given input

Reply:
@justyna-13 Need some answers on this